src/game/Spells/SpellAuras.cpp

```diff
@@ -1285,6 +1285,16 @@ void Aura::HandleAuraDummy(bool apply, bool Real)
                 if (Unit* caster = GetCaster())
                     caster->ModifyHealth(caster->GetMaxHealth() - caster->GetHealth());
                 return;
+            case 25185:                                     // Itch
```

Comment:
This spell has a scripted Effect self-cast, no clue what it does. Maybe it is related to the Catalyst cast, maybe it isn't.